contrib/kafka/kafkaeventsource-operator/deploy/operator.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kafkaeventsource-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: kafkaeventsource-operator
+  template:
+    metadata:
+      labels:
+        name: kafkaeventsource-operator
+    spec:
+      serviceAccountName: kafkaeventsource-operator
+      containers:
+        - name: kafkaeventsource-operator
+          image: docker.io/sjwoodman/kafkaeventsource-operator:latest
+          ports:
+          - containerPort: 60000
+            name: metrics
+          command:
```

Comment:
Couldn't get the operator running properly using the `command` entry here. I ended up deleting `command` entirely from `operator.yaml` and instead relying on `ENTRYPOINT` in the corresponding Dockerfile.

Error message:
```
 Normal   Scheduled  2m                 default-scheduler      Successfully assigned default/kafkaeventsource-operator-7479f8d7d-wmdtj to 10.176.7.123
  Normal   Pulled     1m (x5 over 2m)    kubelet, 10.176.7.123  Container image "docker.io/dubee/kafkaeventsource-operator:latest" already present on machine
  Normal   Created    1m (x5 over 2m)    kubelet, 10.176.7.123  Created container
  Warning  Failed     1m (x5 over 2m)    kubelet, 10.176.7.123  Error: failed to create containerd task: OCI runtime create failed: container_linux.go:344: starting container process caused "exec: \"kafkaeventsource-operator\": executable file not found in $PATH": unknown
```